### PR TITLE
Make compatible with Jakarta namespace

### PR DESCRIPTION
**Pull request by farazhv**
*Issue #, if available:*

Multipart form upload is not supported

*Description of changes:*
Compatible with Jakarta namespace

By submitting this pull request

- [x] I confirm that my contribution is made under the terms of the Apache 2.0 license.
- [x] I confirm that I've made a best effort attempt to update all relevant documentation.

### CODE REVIEW

**Comment by farazhv** [comment]
With reference to issue: https://github.com/awslabs/aws-serverless-java-container/issues/477

File: aws-serverless-java-container-core/src/main/java/com/amazonaws/serverless/proxy/model/AwsProxyRequest.java [hunk 1]
```diff
@@ -32,9 +33,7 @@ public class AwsProxyRequest {
     private String resource;
     private AwsProxyRequestContext requestContext;
     private MultiValuedTreeMap<String, String> multiValueQueryStringParameters;
-    private Map<String, String> queryStringParameters;
```
Comment (deki) [inline aws-serverless-java-container-core/src/main/java/com/amazonaws/serverless/proxy/model/AwsProxyRequest.java:35]: looks like a merge issue to me, this shouldn't be removed

File: aws-serverless-java-container-core/src/main/java/com/amazonaws/serverless/proxy/model/AwsProxyRequest.java [hunk 2]
```diff
@@ -32,9 +33,7 @@ public class AwsProxyRequest {
     private String resource;
     private AwsProxyRequestContext requestContext;
     private MultiValuedTreeMap<String, String> multiValueQueryStringParameters;
-    private Map<String, String> queryStringParameters; 
     private Headers multiValueHeaders;
-    private SingleValueHeaders headers;
```
Comment (deki) [inline aws-serverless-java-container-core/src/main/java/com/amazonaws/serverless/proxy/model/AwsProxyRequest.java:37]: looks like a merge issue to me, this shouldn't be removed

File: aws-serverless-java-container-core/src/main/java/com/amazonaws/serverless/proxy/model/SingleValueHeaders.java [hunk 3]
```diff
@@ -1,24 +0,0 @@
-/*
```
Comment (deki) [inline aws-serverless-java-container-core/src/main/java/com/amazonaws/serverless/proxy/model/SingleValueHeaders.java:1]: looks like a merge issue to me, this shouldn't be removed

**Review by deki — CHANGES_REQUESTED** [review]
Thanks for the work you put into it. 

There are a lot of whitespace changes make it difficult to sort out the real changes but I already saw some places that look like a merge error to me. Would you mind double checking?

I'll probably merge this to a different branch, so another volunteer (or myself) can work on the Multipart stuff. Spring 6 and Spring Boot 3 will be released soon and I will include it along with this change as well.

**Comment by farazhv** [comment]
Thank you for pointing this out. 
 
Since this branch was based on an earlier commit of the master branch, it didn’t include the most recent change. My last commit to this branch addresses this and resolves the merge conflict. 

We are using the submitted code in a lambda to host our webserver – so it is working for us.

**Comment by tntsoft** [comment]
@deki Is there any timeline in place to get these changes sorted, merged and released? Got a Spring Boot 3 application which I'm planning to move to AWS Lambda, so am wondering if I need to go and tinker it myself, or put it on the back burner until the new release.

**Comment by deki** [comment]
Hi folks,
sorry for the delay but we finally made the changes in the main branch.

@farazhv while comparing with your PR, I notice a NullInputStream and some changes related to binary reading? Is that needed? Could you please test the main branch if it's working for you?

**Comment by farazhv** [comment]
@deki sure we will test the main branch as soon as we can

**Comment by farazhv** [comment]
@deki we aren't blocked by this so we won't be able to test it for the time being

**Pull request closed without merging**